apps/src/applab/commands.js

```diff
@@ -944,7 +944,11 @@ applabCommands.setImageURL = function (opts) {
 
   var element = document.getElementById(opts.elementId);
   if (divApplab.contains(element) && element.tagName === 'IMG') {
-    element.src = assetPrefix.fixPath(opts.src);
+    if (ICON_PREFIX_REGEX.test(opts.src)) {
+      element.src = assetPrefix.renderIconToString(opts.src, element);
```

Comment:
`element.width` and `element.height` will stay the same?  (I think yes, but wanted to double check.)